flasgger/utils.py

```diff
@@ -579,6 +591,10 @@ def extract_definitions(alist, level=None, endpoint=None, verb=None,
     We require an 'id' field for the schema to be correctly
     added to the definitions list.
     """
+    defs = list()
+    if alist is None:
+        return defs
```

Comment:
I added this short circuit as it does saves computational cost by returning early and reduces the indentation of code previously nested in this _if_